.gitignore

```diff
@@ -1,6 +1,6 @@
 /node_modules
+/assets/demo
+/assets/main
 *.orig
 /.tmp
-/dist
-/assets
```

Comment:
These were in here for keeping the built files out of master.  The two added lines should probably get added to the `gh-pages` branch where we release from instead